docs/http_api.rst

```diff
@@ -113,10 +113,7 @@ type
     The string representing the type of permission.
 object_id
     A string that uniquely identifies the object that this permission acts
-    upon. "null" if this permission does not act on a specific object.
-metadata
-    A flat object that can be used to add any additional information that
-    might be needed for the permission.
+    upon. Empty string if this permission does not act on a specific object.
```

Comment:
We can, although the django docs suggest against it https://docs.djangoproject.com/en/1.9/ref/models/fields/#null